pkgs/applications/science/logic/formula/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, buildDotnetModule, dotnet-sdk_5 }:
+
+buildDotnetModule rec {
+  pname = "formula-dotnet";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "VUISIS";
+    repo = "formula-dotnet";
+    rev = "60dd3d91d931a795fa9fe6b6054f63e03cec9d28";
+    sha256 = "sha256-7odYV4FtImzX3vMUfO7sPlXkw6mOkZOFbnQLpJDXJc4=";
+  };
+
+  nugetDeps = ./nuget.nix;
+  projectFile = "Src/CommandLine/CommandLine.csproj";
+
+  makeWrapperArgs = "--set PATH ${dotnet-sdk_5}/bin";
```

Comment:
This flag isn't used currently, since we manually wrap the program. I don't think it should be required anyway, but if it is you can do something like this when wrapping:

```bash
makeWrapper ... \
  ''${makeWrapperArgs[@]}
```

Or just set it manually of course :)